### ISSUE TITLE
Reduce the environment load time

### ISSUE THREAD

**Issue by PaulDuvall** [stelligent/nando_automation_demo#12]
The current environment load time is about 20 mins. Would like to see if we can run in less time. If we can run two environment at the same time, this becomes a lower priority. 

Priority for demo: Medium or Low if ability to run dual environments.

**Comment by nand0p** [comment]
this probably takes a while unnecessarily

```
$ nl  ../../nando_automation_demo/cloudformation.json |grep "yum -y update"
   573                  "command": "yum -y update"
```

i think this was resolved by @NickDeClario ... dont forget to comment... best practices demo :)

**Comment by nand0p** [comment]
- depends on: https://github.com/stelligent/nando_automation_demo/issues/2
  once we have vpc stacks running in diff regions @trikosuave can have go.py select where to launch,...thereby reducing launch time of vpc resouces
- depends on: https://github.com/stelligent/nando_automation_demo/issues/21
  once we bypass the timeconsuming elasticbeanstalk aws launched substack, and access ecs directly by @trikosuave go.py script, this will greatly reduce launch time

**Comment by trikosuave** [comment]
This is good to go. A full stack is 30 minutes. Consecutive stacks will re use vpc/sg/rds and will take 15 minutes to build.
